src/golang/lib/workflow/artifact/response.go

```diff
@@ -28,30 +34,34 @@ type ResultResponse struct {
 }
 
 func NewResultResponseFromDbObjects(
-	DbArtifact *artifact.DBArtifact,
-	DbArtifactResult *artifact_result.ArtifactResult,
+	dbArtifact *artifact.DBArtifact,
```

Comment:
What happened in this file? Just casing adjustments and linter fixes?

Reply:
lol yes variable names should be lower-case, I forgot that in previous PR